lazy_theta_star_planner/include/lazy_theta_star_planner/lazy_theta_star2.h

```diff
@@ -0,0 +1,136 @@
+// Copyright 2020 Anshumaan Singh
```

Comment:
Please rename this file to not just be `2`, also all the headers in C++ should be `hpp`